src/components/sidebar/left-sidebar/CountryList.tsx

```diff
@@ -1,9 +1,32 @@
-import React from "react";
+import React, { useContext } from "react";
+import { AppContext } from "../../../context";
+import { getAggregateData } from "../../../metaAnalysis";
+import './CountryList.css';
+import _ from "lodash";
 
 export default function CountryList() {
+  const [state, dispatch] = useContext(AppContext);
+
+  const aggregatedData = _.sortBy(getAggregateData(state.filtered_records, 'country'), ['seroprevalence']).reverse();

```

Comment:
Not sure if we'd rather sort countries by alphabetical order or seroprevalence as you've done here. Maybe a question to bring up during a design review